RootSource.cc

```diff
@@ -38,19 +40,19 @@ RootSource::RootSource(std::string const& iName) :
 			       &delayedReader_);
     branches_.emplace_back(b);
     if(eventAuxiliaryBranchName == dataProducts_.back().name()) {
-      eventAuxReader_ = EventAuxReader(dataProducts_.back().address());
+      aux_branch = dataProducts_.back().address();
     }
   }
-  if(not eventAuxReader_) {
-    eventAuxReader_ = EventAuxReader(nullptr);
+  if(aux_branch) {
+    eventAuxReader_.bindToBranch(aux_branch);
```

Comment:
Where does `eventAuxReader_` in this case figure out which file type it is using? It appears to be defaultly constructed.

Reply:
This code is broken; I will adjust.